cmd/ui/src/App.tsx

```diff
@@ -108,10 +111,20 @@ const Inner: React.FC = () => {
         }
     }, [dispatch, authState.isInitialized]);
 
-    // prefetch feature flags
+    // initialize theme
     useEffect(() => {
-        queryClient.prefetchQuery(featureFlagKeys.all, getFeatureFlags);
-    }, [queryClient]);
+        const theme = darkMode ? 'dark' : 'light';
+        document.body.setAttribute('class', theme);
+
+        // TODO: Consider adding flag keys to cue files so we don't use random strings. Consider adding more flexibility/composability to side effects for toggling feature flags on and off
```

Comment:
I have ideas on how to do this, however it would add  noise to this PR and the dark_mode changes as a whole